app/Interactive.hs

```diff
@@ -40,8 +40,9 @@ mainLoop func = do
           H.outputStrLn =<< liftIO (func inp)
           mainLoop func
 
+-- TODO: remove
 parseString :: String -> Maybe (HR.Term Nat.Ty Nat.Val)
-parseString = HR.generateParser Nat.t
+parseString = undefined
```

Comment:
Should this be removed, or is it still mentioned somewhere else?

Reply:
A quick grep shows that it isn't used anymore and can be removed, however I think it is outside the scope of this PR.

Reply:
oh sorry, I didn't notice the filename, never mind